### DISCUSSION TITLE
stoicism/provisions/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#17] [General]
# stoicism/provisions/

https://shanenull.com/stoicism/provisions/

**Comment by shane0** [comment]
# snow peak

- https://www.snowpeak.com/pages/care-guide

**Reply by shane0** [reply]
https://www.snowpeak.com/collections/summer-festival-2024

**Reply by shane0** [reply]
- collaborate with polishing syndicate
- https://www.migaki.com/

**Reply by shane0** [reply]
https://cdn.shopify.com/s/files/1/0330/3299/4860/files/ES-141_manual_A4_web.pdf?v=1720032588

**Reply by shane0** [reply]
https://www.snowpeak.com/blogs/explore/how-to-takibi-time

**Reply by shane0** [reply]
https://www.nippon.com/en/guide-to-japan/gu020002/

https://www.snowpeak.com/products/lounge-cushion

**Reply by shane0** [reply]
# sitting

- https://www.snowpeak.com/blogs/explore/getting-started-with-igt

**Reply by shane0** [reply]
https://goodsofjapan.com/products/snow-peak-makita-mkt-102-field-fan-with-ac-adapter-and-battery-charger-sold-separately

**Reply by shane0** [reply]
https://www.migaki.com/

**Reply by shane0** [reply]
https://korogi.co/products/bamboo-folding-2u-igt-frame-for-50l-shelcon

**Reply by shane0** [reply]
## furoshiki

- https://youtu.be/cy1OInDW43g

**Reply by shane0** [reply]
Tarp https://youtu.be/O_oWDVE1Sj8?si=_OXAv5T82Lgko58e

**Comment by shane0** [comment]
https://www.bostongeneralstore.com/

https://www.olympiaprovisions.com/

https://topdrawershop.com/

https://dehen1920.com/

**Reply by shane0** [reply]
Abraham Moon & Sons

**Reply by shane0** [reply]
Halley Stevensons evermax

**Reply by shane0** [reply]
https://blancheandmimi.com/

**Reply by shane0** [reply]
https://merchantandrhoades.com/collections/frontpage

**Reply by shane0** [reply]
https://korogi.co/collections/igt-accessories

**Comment by shane0** [comment]
# japan

```text
UNIFLAME ユニフレーム キッチンスタンド3 ＆ 収納ケース 2点セット (611715＆610411)
```

- [amazon jp](https://www.amazon.co.jp/UNIFLAME-%E3%83%A6%E3%83%8B%E3%83%95%E3%83%AC%E3%83%BC%E3%83%A0-%E3%82%AD%E3%83%83%E3%83%81%E3%83%B3%E3%82%B9%E3%82%BF%E3%83%B3%E3%83%893-%E5%8F%8E%E7%B4%8D%E3%82%B1%E3%83%BC%E3%82%B9-611715%EF%BC%86610411/dp/B098WRLYQ2/ref=sr_1_55)

**Reply by shane0** [reply]
- wallet https://hightidestoredtla.com/collections/cand-design-works?srsltid=AfmBOopuc5efSinvHqGNu_aXQLgSUWqsLgBxRExAGBpUX3O6xOYpqLjz

**Reply by shane0** [reply]
https://www.youtube.com/watch?v=ZtXum-ML4Gc

**Reply by shane0** [reply]
https://www.akazuki.com/

**Reply by shane0** [reply]
Onoe MY-7204 Galvanized Hot Water Bottle (Camp Disaster Preparedness Goods, Metal, Made in Japan, Capacity: 0.7 gal (2.4 L)

**Reply by shane0** [reply]
https://www.youtube.com/watch?v=ZtXum-ML4Gc

**Reply by shane0** [reply]
https://youtu.be/vKy-Rt4TrNc?si=MQZ-_2mQlWYD1s5m

**Reply by shane0** [reply]
https://youtu.be/VChUHSmCJrU?si=_vlcmZd1Frjcv-_S

**Reply by shane0** [reply]
https://www.newton.com.tw/

**Reply by shane0** [reply]
https://youtu.be/iRqOiZNstGk?si=v2fmjeWsxCjlwbML

**Reply by shane0** [reply]
# funny tshirts

- https://www.tokyo-tiger.com/products/tokyo-tiger-sayagata-buddha-graphic-washed-t-shirt
- https://www.tokyo-tiger.com/products/tokyo-tiger-stupid-possum-japan-classic-t-shirt
-https://www.tokyo-tiger.com/products/tokyo-tiger-born-to-shit-forced-to-wipe-washed-t-shirt
- https://www.tokyo-tiger.com/products/tokyo-tiger-frogs-trapped-in-the-void-washed-t-shirt
- https://www.tokyo-tiger.com/products/tokyo-tiger-ukiyoe-waves-japan-washed-t-shirt

**Reply by shane0** [reply]
https://youtu.be/BdzwVsDTB0c?si=SnekrbXYG1Gk_7UI

**Reply by shane0** [reply]
```text
japanese handmade braided wire cuff bracelets
```

**Reply by shane0** [reply]
go stone https://youtu.be/whvAB7Jl7fc?si=o0cp8_y1yy1wd2Oj

**Reply by shane0** [reply]
https://www.amazon.co.jp/s?k=porter+yoshida

**Reply by shane0** [reply]
# lamps

https://youtu.be/xzRHSqKYiVE?si=jE7hIjOTe7q5HZ0d

**Reply by shane0** [reply]
https://korogi.co/collections/lighting

**Reply by shane0** [reply]
- [book end](https://stackedstore.com/products/detail-slope-bookend?variant=40457303097408)

**Comment by shane0** [comment]
https://shanenull.com/food/mocktails/

**Comment by shane0** [comment]
# planner

- OHTO Wooden Mini Mechanical Pencil with Clip
- traveler passport 2

**Reply by shane0** [reply]
- Leather Cover - H 5.3"x W 3.9" x D 0.4" 
- Notebook - H 4.9" x W 3.5" x D 0.2", Blank, 64 pages
- The ratio of height to width for a 4.9" x 3.5" image is approximately 1.4:1.

**Reply by shane0** [reply]
https://www.bostongeneralstore.com/collections/travelers-notebook-collection/products/travelers-passport-notebook

**Reply by shane0** [reply]
https://www.youtube.com/watch?v=ic1UUSWOuSg

**Reply by shane0** [reply]
# nahe

- https://hightidestoredtla.com/products/2025-weekly-planner-nahe-a5-vertical

**Reply by shane0** [reply]
https://hightidestoredtla.com/products/adjustable-book-band-wide-a

**Reply by shane0** [reply]
https://hightidestoredtla.com/products/gadget-rest-stand

**Comment by shane0** [comment]
# watches

- https://calibercorner.com/eta-caliber-g10-212/
- indeglo battery?
- coros 2
- apple ?

**Reply by shane0** [reply]
set the tissot https://youtu.be/QU26DjH3M9I

**Reply by shane0** [reply]
https://youtu.be/qrusqiAo4nE

**Reply by shane0** [reply]
394 battery

**Reply by shane0** [reply]
T055417A
h8g 8pwbjb
19jaqq
https://www.tissotwatches.com/en-th/t0554171104700.html

**Reply by shane0** [reply]
```text
Tissot PRC 200 Chronograph
T055.417.11.047.00
Diameter:42 mm
316L stainless steel case
Screw-down crown and caseback
```

**Reply by shane0** [reply]
T605031423 strap reference
19mm

**Reply by shane0** [reply]
https://www.farrar-tanner.com/executive-desk-clock-hygrometer-thermometer-set-with-stand

**Comment by shane0** [comment]
# pairing

- https://www.holts.com/clubhouse/pairings
- breath: carbonated + brush + mouthwash + hydrogen peroxide

**Reply by shane0** [reply]
Lane 1Q

**Comment by shane0** [comment]
# cool

- https://szphoton.com/products/fot?variant=46766600716610

**Reply by shane0** [reply]
https://www.youtube.com/watch?v=km0jx-AMLGA&t=360s

**Reply by shane0** [reply]
https://store.mudita.com/store/alarm-clocks
https://hightidestoredtla.com/collections/penco/clock

**Reply by shane0** [reply]
https://www.taylorstitch.com/collections/the-carnegie-pant

**Reply by shane0** [reply]
# leather kit bag

- some kind of brushes and scrapeers for cleaning  boots and sneakers

**Reply by shane0** [reply]
# gear repair

- need a bag to organize this stuff
- mainly camping gear repair
- also daily clothing

**Reply by shane0** [reply]
# cast iron

- need the cast iron scrapers
- and furoshiki

**Reply by shane0** [reply]
fa wu stamp

https://www.etsy.com/listing/1741164004/custom-pottery-stamp-personalized

**Reply by shane0** [reply]
https://inovativ.com/pages/voyager

**Reply by shane0** [reply]
https://www.youtube.com/@MurdockLondon

**Reply by shane0** [reply]
# beard

| Brand | Location | Products | Description |
|---|---|---|---|
| **Bearded Oregon** | Salem, OR | Beard oils, balms, butters | Handcrafted premium beard products formulated with unique blends of Oregon-sourced ingredients. citeturn0search0 |
| **Northwest Beard Supply** | Central OR | Beard oils | Family-owned, small-batch company designing all-natural beard products to enhance comfort, style, and scent experience. citeturn0search19 |
| **Beardwood Oil** | Bend, OR | Beard oils, balms | Premium beard care products made with organic, high-quality oils and balms to soften and maintain healthy beards. citeturn0search6 |
| **Bigfoot Beard Products** | Lacey, WA | Beard balms, oils, waxes, shampoos | Handcrafted in the Pacific Northwest, offering a unique line of natural products exclusive to men. citeturn0search1 |
| **Seattle Pomade Co.** | Seattle, WA | Beard oils | Certified organic beard oil promoting growth for soft and itch-free facial hair. citeturn0search12 |
hese brands emphasize natural ingredients and local craftsmanship, providing a range of products to suit various beard care needs.

**Reply by shane0** [reply]
https://ltwrightknives.com/products/wild-harvest-les-stroud-foraging-tool

**Reply by shane0** [reply]
Heavyweight four-end jersey knit

**Reply by shane0** [reply]
https://panslings.com/

**Comment by shane0** [comment]
# france

- https://www.instagram.com/reel/DDcL2p6snmA/
